pom.xml

```diff
@@ -212,35 +212,45 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-enforcer-plugin</artifactId>
-        <configuration>
-          <rules>
-            <!-- The following dependencies are hazardous for builds. -->
-            <bannedDependencies>
-              <excludes>
-                <!-- clashes with commons-logging:commons-logging -->
-                <exclude>commons-logging:commons-logging-api</exclude>
-                <!-- junit repackages hamcrest into its own jar, which is a big no-no.
-                     junit-dep fixes this problem -->
-                <exclude>junit:junit-dep</exclude>
-                <exclude>junit:junit:[,4.11)</exclude>
-                <!-- completely and utterly broken -->
-                <exclude>org.weakref:jmxutils:(,1.11]</exclude>
-                <!-- use guava -->
-                <exclude>com.google.collections:google-collections</exclude>
-                <!-- but not the badly numbered ones... -->
-                <exclude>com.google.guava:guava</exclude>
-                <!-- 1.8 and 1.9 are generified badly -->
-                <exclude>commons-configuration:commons-configuration:[1.8,1.9]</exclude>
-                <!-- ban all versions of asm that do not work on JDK8 -->
-                <exclude>org.ow2.asm:asm:[,5.0.2)</exclude>
-              </excludes>
-              <includes>
-                <!-- whitelist the well numbered guava releases -->
-                <include>com.google.guava:guava:[10.0.1,)</include>
-              </includes>
-            </bannedDependencies>
-          </rules>
-        </configuration>
+        <executions>
+          <execution>
+            <id>basepom-banned-dependencies</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <skip>${basepom.check.skip-enforcer}</skip>
```

Comment:
I copied these from [here](https://github.com/basepom/basepom/blob/31e7e9acafec12878a1bb526298e32d4e9bc0cc1/foundation/pom.xml#L290-L291). Looking again, it should e safe to omit them, though being explicit here might be nice.

Reply:
I think it's worth keeping them